src/App.tsx

```diff
@@ -20,12 +22,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={e => setMovieTitle(e.target.value)}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesFromServer} movieTitle={movieTitle} />
```

Comment:
Ensure that the `MoviesList` component is designed to handle the `movieTitle` prop for filtering or searching movies. If this functionality is not implemented in `MoviesList`, the search feature will not work as intended.